metricbeat/include/list_common.go

```diff
@@ -128,6 +128,7 @@ import (
 	_ "github.com/elastic/beats/metricbeat/module/redis/info"
 	_ "github.com/elastic/beats/metricbeat/module/redis/key"
 	_ "github.com/elastic/beats/metricbeat/module/redis/keyspace"
+	_ "github.com/elastic/beats/metricbeat/module/stan"
```

Comment:
It seems that this shouldn't be here. It will be fixed with a `make update` command inside `/beats/metricbeat` folder.

Reply:
Unsure why running `make check` in my local repo doesn't cause this error though while Travis complains? Been driving me nuts

Reply:
`make update` adds it _back_ after I removed it and ran locally. Something is out of sync and even merging the PR into a local copy of `master` and running these updates don't reproduce the error for myself either.